spec/services/between_categories/creator_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe 'BetweenCategoriesTransactions::Creator', type: :feature do
```

Comment:
Why not use more simple `describe BetweenCategoriesTransactions::Creator do`?

Reply:
@sinpai Why is it bad?

Reply:
@marderer it's not bad, on the other hand we can always use simpler format if possible ;)